web/templates/settings/admin_human_form.hbs

```diff
@@ -34,11 +34,13 @@
             </button>
         </span>
         {{else}}
+        {{#unless is_deleted}}
```

Comment:
if we can't access deleted user in any way in web UI then what's the need for this ?

Reply:
This can be reached via edit user in Settings/users in organisation settings.